packages/desktop-client/src/components/schedules/SchedulesTable.tsx

```
@@ -267,94 +275,104 @@ export function SchedulesTable({
+              : null}
+          </Field>
+          <Field width={120} name="status" style={{ alignItems: 'flex-start' }}>
+            {status && <StatusBadge status={status} />}
           </Field>
+          <ScheduleAmountCell
+            amount={schedule._amount}
+            op={schedule._amountOp}
+          />
+          {!minimal && (
+            <Field width={80} style={{ textAlign: 'center' }}>
+              {schedule._date && schedule._date.frequency && (
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)